spec/suma/api/commerce_spec.rb

```diff
@@ -391,13 +378,6 @@
       expect(checkout.refresh).to have_attributes(fulfillment_option: be === opt)
     end
 
-    it "errors if a nil fulfillment empty is passed" do
```

Comment:
Why was this removed?

Reply:
Error on my part. I updated the test with "..allows nil fulfillment option".